lib/globalconstraintregistrar.js

```diff
@@ -80,46 +88,50 @@ export class GlobalConstraintRegistrar extends ExtendTruffleContract(SolidityGlo
       options.avatar,
       options.globalConstraint,
       options.globalConstraintParametersHash,
-      options.votingMachineHash);
+      options.votingMachineHash
+    );
 
     return tx;
   }
 
-  async proposeToRemoveGlobalConstraint(opts={}) {
-
+  async proposeToRemoveGlobalConstraint(opts = {}) {
     const defaults = {
       /**
        * avatar address
        */
-      avatar: undefined
+      avatar: undefined,
       /**
        *  the address of the global constraint to remove
        */
-      , globalConstraint: undefined
+      globalConstraint: undefined
     };
 
     const options = dopts(opts, defaults, { allowUnknown: true });
 
-    if (!options.avatar)
-    {
+    if (!options.avatar) {
       throw new Error("avatar address is not defined");
     }
 
-    if (!options.globalConstraint)
-    {
+    if (!options.globalConstraint) {
       throw new Error("avatar globalConstraint is not defined");
     }
 
-    const tx = await this.contract.proposeToRemoveGC(options.avatar, options.globalConstraint);
+    const tx = await this.contract.proposeToRemoveGC(
+      options.avatar,
+      options.globalConstraint
+    );
 
     return tx;
   }
 
   async setParams(params) {
-    return await this._setParameters(params.voteParametersHash, params.votingMachine);
+    return await this._setParameters(
+      params.voteParametersHash,
+      params.votingMachine
+    );
   }
 
   getDefaultPermissions(overrideValue) {
-    return overrideValue || '0x00000007';
+    return overrideValue || "0x00000007";
```

Comment:
Why this scheme as default value different from contributionreward default value ?